interface/reports/daily_summary_report.php

```diff
@@ -67,7 +67,7 @@ function submitForm() {
                 }
             }
 
-            $( document ).ready(function(){
+            $(function(){
                 $('.datepicker').datetimepicker({
```

Comment:
it's only syntax but is one version more accepted than the other?
the one in this script with no space btw function and parens
```
$(function(){
```
or in previous script
```
$(function () {
```
guessing no space

Reply:
I suggest

$(function() {

});